File: SeperatedInterface/domain/customer.py
"""
The Customer can place an order.
"""
from order import Order

class Customer():
	__slots__ = ('name', 'weight')

	def __init__(self, name, weight):
		self.name = name
		self.weight = wieght

	def order(self, food):
		order = Order(food, self.name, self.weight)
		order.save()

File: SeperatedInterface/src/domain/Customer.java
package domain;
import mapper.OrderMapper;

public class Customer {

	String name;
	
	public Customer(String name){
		this.name = name;
	}
	
	public void placeOrder(String item){
		Order order = new OrderMapper()
			.setItem(item)
			.setName(this.name);
		order.save();
	}
	
}
